viz.yaml

```diff
@@ -0,0 +1,81 @@
+vizlab: "0.3.6"
+info:
+  id: gage-conditions
+  name: USGS Gage Conditions
+  date: 2018-06-11
+  publish-date: 2019-06-25
+  path: /gage-conditions
+  analytics-id: # WE WILL NEED TO FILL THIS IN AT SOME POINT
+  description: Map of all USGS gages and their conditions.
+  required-packages:
+    vizlab:
+      repo: github
+      version: 0.3.6
+      name: USGS-VIZLAB/vizlab
+      ref: v0.3.6
+    dataRetrieval:
+      repo: CRAN
+      version: 2.7.4
+    dplyr:
+      repo: CRAN
+      version: 0.7.4
+    tidyr:
+      repo: CRAN
+      version: 0.8.1
+  contributors:
+    -
+      name: Lindsay Carr
+      email: lcarr@usgs.gov
+      affiliation: U.S. Geological Survey
+    -
+      name: Jordan Read
+      email: jread@usgs.gov
+      affiliation: U.S. Geological Survey
+parameter:
+  - id: plot_monthday
+    monthday: "05-01"
+  - id: plot_year
+    year: 2018
+  - id: block_size
+    size: 10
+  - id: discharge_percentiles
+    percentiles: ["05","10","20","25","50","75","80","90","95"]
+fetch:
+  - id: dv_sites
+    location: cache/dv_sites.json
+    reader: json
+    fetcher: dv_sites
+    scripts: [scripts/fetch/dv_sites.R]
+    depends:
+      year: plot_year
+  - id: site_stats # took about 20 min for all time
```

Comment:
I think it is good to get a big data pull like this out of the way once, but I do think it should depend on plot_year and potentially only fetch the plot year (if that is possible). Reason being that a stats pull done in 2018 will be out of date for 2019 use. My understanding is that the stats are updated as new data come in, so we'd need a new pull if we changed the plot_year to 2019 anyhow.

Reply:
I'm a little confused about the stat service now. I was getting lots of errors when I tried to add dates (started a convo in dataRetrieval) and it seems like it's because there is not stat data available for every year of data:

Start date 2018 gives no data: https://waterservices.usgs.gov/nwis/stat/?sites=50010500&statType=P05,P10,P20,P25,P50,P75,P80,P90,P95&statReportType=daily&parameterCd=00060&startDT=2018-01-01

No dates gives data only up until 2002: https://waterservices.usgs.gov/nwis/stat/?sites=50010500&statType=P05,P10,P20,P25,P50,P75,P80,P90,P95&statReportType=daily&parameterCd=00060

Reply:
This is blocking me from being able to work on the webpack stuff. So, I made a separate issue for it and we can discuss there https://github.com/USGS-VIZLAB/gage-conditions/issues/3